src/sv_user.c

```diff
@@ -3597,6 +3669,12 @@ void SV_RunCmd (usercmd_t *ucmd, qbool inside, qbool second_attempt) //bliP: 24/
 	//<-
 	cmd = *ucmd;
 
+	// Apply safestrafe before physics processing
+	if (!inside) {
+		SV_ApplySafestrafe(sv_client, ucmd);
+		cmd = *ucmd; // Update cmd with potentially modified ucmd
```

Comment:
minor nitpick, why not move the `cmd = *cmd;` above the `if ( ...) { .. }` block to under it to just do it once.

Reply:
> minor nitpick, why not move the `cmd = *cmd;` above the `if ( ...) { .. }` block to under it to just do it once.

@dusty-qw, can you tackle this?